index.html

```diff
@@ -169,11 +204,12 @@ <h1>The Registration Process</h1>
       <li>
 Any addition to the DID Core Registries MUST link, via at least a URL,
 preferably a content-integrity protected one, to the defining specification so
-that implementers can implement the property.
+that implementers can implement the property or value.
       </li>
       <li>
 Any addition to the DID Core Registries that is a property or value, MUST
-specify a machine readable JSON-LD Context for the addition.
+specify a machine readable JSON-LD Context for the addition via a URL, 
+preferably a content-integrity protected one.
```

Comment:
We're going to end up removing this language... no one is submitting content-integrity protected URLs for definitions.

Reply:
agreed, just repeating it to reduce the likelihood of objection.

Reply:
resolving, we can remove this all at once later.